docs/summary.md

```diff
@@ -19,7 +19,6 @@
 * [Prediction monitoring](deployments/prediction-monitoring.md)
 * [Compute](deployments/compute.md)
 * [API statuses](deployments/statuses.md)
-* [Python client](deployments/python-client.md)
```

Comment:
Should we also remove `python-client.md` for now? We could link this commit in the ticket you just made so we could get the language back when we add it back